rivershared/riverpilot/pilot.go

```diff
@@ -20,8 +20,8 @@ type Pilot interface {
 	PilotPeriodicJob
 
 	// JobCleanerQueuesExcluded returns queues that should be excluded from the
-	// main River client's JobCleaner. Empty omitted queues should return nil as
-	// instead of empty array.
+	// main River client's JobCleaner. If no queues should be omitted, this
+	// function should return nil as opposed to an empty array.
```

Comment:
What's the difference in behavior if I _do_ return an empty array? Maybe worth spelling out if anything but I would expect any logic depending on this to consider them equivalent?